src/index.js

```diff
@@ -1,5 +1,35 @@
 import './style.css';
+import { showData } from '../modules/showScore.js';
+import displayScores from '../modules/scores.js';
 
-import showScores from '../modules/showScore.js';
+const refreshBtn = document.querySelector('.refresh-btn');
+const submitBtn = document.querySelector('#submit-btn');
+const player = document.getElementById('name');
+const score = document.getElementById('score');
 
-document.addEventListener('DOMContentLoaded', showScores);
\ No newline at end of file
+window.addEventListener('DOMContentLoaded', () => {
+  displayScores();
+
+  refreshBtn.addEventListener('click', () => {
+    displayScores();
+  });
+
+  submitBtn.addEventListener('click', () => {
+    if (player.value.length === 0) {
+      return;
+    }
+
+    //   if (!score.value.match(regex2)) {
+    //     errorMessage.style.display = "block";
+    //     errorMessage.style.color = "yellow";
+    //     errorMessage.style.fontSize = "12px";
+    //     errorMessage.style.lineHeight = "24px";
+    //     return;
+    //   }
+    //   errorMessage.style.display = "none";
+
```

Comment:
- [ ] Great job, to keep your code clean, please don't leave commented code.

Reply:
@Wes-Isaac Thanks for the feedback